Tests/SentryTests/Helper/SentryFileManagerTests.swift

```diff
@@ -9,7 +9,7 @@ class SentryFileManagerTests: XCTestCase {
     override func setUp() {
         super.setUp()
         do {
-            sut = try SentryFileManager(dsn: TestConstants.dsn)
+            sut = try SentryFileManager(dsn: TestConstants.dsn, with: TestCurrentDateProvider())
```

Comment:
H: more like a question, but why this is called `with` and not `withCurrentDateProvider` or `currentDateProvider` by swift? `with` is not specific enough and misleading, can we fix that?

Reply:
That's the default from Swift. In Objective-C, it's pretty common to have verbose naming for method parameters and also constructor parameters. Swift is less verbose so they just cut out some things for you when calling Objective-C code. Now that I again think about it it's more common to start the name of the second constructor argument with `and`.

Reply:
mm but that's weird, and if you have 2 params andA, andB, how does that work? if it's a swift thingie, consider this resolved, just curious now

Reply:
In Objective-C, you usually do `initWithSomething: andAntoherThing: andSomethingElse: ` or you do `initWithSomething anotherThing: somethingElse: `. In Swift you usually to `init(something: , anotherThing: , somethingElse: )`

Reply:
yeah I got it, but we have `withCurrentDateProvider` in ObjC and this has been translated to `with` on Swift, its weird for me, I'd like to change that, if its possible